processor/stream/processor.go

```diff
@@ -165,15 +137,15 @@ func (p *Processor) readMetadata(reqMeta map[string]interface{}, reader StreamRe
 }
 
 // HandleRawModel validates and decodes a single json object into its struct form
-func (p *Processor) HandleRawModel(rawModel map[string]interface{}) (transform.Transformable, error) {
+func HandleRawModel(rawModel map[string]interface{}, models []EventModel, requestTime time.Time, metadata metadata.Metadata) (transform.Transformable, error) {
```

Comment:
Why is this exposed more than it was already? Can we unexport it?

Reply:
yeah, definitely